index.html

```diff
@@ -1,47 +1,54 @@
 <html>
   <head>
+    <link rel="stylesheet" type="text/css" href="/style.css"  />
     <script src="https://web-app.sandbox.token.io/token.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
   </head>
   <body>
-    <div id="tokenPayBtn" style="width:200px"></div>
+    <button id="tokenPayBtn">Token Quick Checkout</button>
 
     <script>
```

Comment:
We should consider moving this into a sepparate js file like we do on merchant-sample-java